ergo-wallet/src/main/scala/org/ergoplatform/wallet/interpreter/ErgoInterpreter.scala

```diff
@@ -82,12 +87,62 @@ class ErgoInterpreter(params: ErgoLikeParameters)(implicit IR: IRContext)
 
 object ErgoInterpreter {
 
+  /** Creates an interpreter with the given parameters, [[RuntimeIRContext]] for script processing and
+    * the `scriptProcessor` for handling compilation caching and handling pre-compiled scripts.
+    */
   def apply(params: ErgoLikeParameters): ErgoInterpreter =
-    new ErgoInterpreter(params)(new RuntimeIRContext)
+    new ErgoInterpreter(params)(new RuntimeIRContext) {
+      override val precompiledScriptProcessor = scriptProcessor
```

Comment:
Type annotation is needed for public member

Reply:
fixed